libs/utilities/src/lib/axios-error/axios-error.ts

```diff
@@ -0,0 +1,6 @@
+import { AxiosError } from 'axios';
+
+export const errorToAxiosError = (
+  error: unknown,
+): AxiosError<{ [key: string]: string }> =>
+  error as AxiosError<{ [key: string]: string }>;
```

Comment:
We should be more declarative on the possible expected data types. Then you won't need to declare `response.data['reason']` and can use `response.data.reason` instead. Added a bunch of suggestions to help make the cleanup on this faster 👍 

```suggestion
) => error as AxiosError<{ reason: string; status: string; }>;
```